Src/jpdf.cpp

```diff
@@ -585,6 +616,7 @@ main (int   argc,
                     }
 
                     // Now divide by volume to make pdf integral to unity (has to come second)
+                    // Is this correct, if conditioning is applied? In that case sum(bin)!=domainVol
```

Comment:
No, this is not correct. There needs to be an additional value being carried around that tracks the volume being integrated over. Qualitatively it won't affect anything though.